libraries/session/src/main/java/androidx/media3/session/MediaButtonReceiver.java

```diff
@@ -123,7 +123,7 @@ public class MediaButtonReceiver extends BroadcastReceiver {
    * strongly recommended way to handle the intent is using the default implementation.
    */
   @Override
-  public void onReceive(Context context, @Nullable Intent intent) {
+  public void onReceive(Context context, Intent intent) {
```

Comment:
As far as I can tell, if there is no @Nullable or @NotNull annotation on Android framework code, then there is no nullability information available and we can't assume this is non-null. 

Agreed the doc is clear about it and we can't do something sensible without an `Intent`. Mind adding `checkNotNull(intent)` when passing it further down. Sorry for being overly picky, I see we would have a NPE right after but I think I'd prefer this being consistently where we get the garbage input. Shouldn't hurt as we agree this doesn't happen.